lerobot/common/policies/diffusion/configuration_diffusion.py

```diff
@@ -110,6 +110,7 @@ class DiffusionConfig:
     diffusion_step_embed_dim: int = 128
     use_film_scale_modulation: bool = True
     # Noise scheduler.
+    noise_scheduler_type: str = 'DDPM'
```

Comment:
Could you please add a dosctring argument for this (preserving ordering)?

Reply:
Done